### PR TITLE
Fixes adapter breadcrumbs link + typo in 'No history' message in programs detailed view

### PR DESCRIPTION
**Pull request by ajainarayanan**

### CODE REVIEW

File: cdap-ui/app/directives/program-history/program-history.html [hunk 1]
```diff
@@ -44,6 +44,6 @@
 
 <div ng-if="runs.length === 0">
   <div class="well well-lg text-center">
-    <div> No History so far.</div>
+    <h3> No history so far.</h3>
   </div>
 </div>
```
Comment (elfenheart) [inline cdap-ui/app/directives/program-history/program-history.html]: Can we remove this outer <div> and add the ng-if to the *well*?

File: cdap-ui/app/features/adapters/routes.js [hunk 2]
```diff
@@ -97,7 +97,8 @@ angular.module(PKG.name + '.feature.adapters')
           templateUrl: 'data-list/data-list.html',
           controller: 'AdapterDatasetsController',
           ncyBreadcrumb: {
-            label: 'History'
+            label: 'History',
```
Comment (elfenheart) [inline cdap-ui/app/features/adapters/routes.js]: label should say **data**?

File: cdap-ui/app/features/adapters/routes.js [hunk 3]
```diff
@@ -97,7 +97,8 @@ angular.module(PKG.name + '.feature.adapters')
...
+            parent: 'adapters.detail.runs'
```
Comment (elfenheart) [inline cdap-ui/app/features/adapters/routes.js]: In other programs, we are not defining this parent..

**Comment by ajainarayanan** [comment]
breadcrumbs

**Comment by elfenheart** [comment]
:+1: LGTM

**Pull request merged**